jobs/haproxy/spec

```diff
@@ -553,6 +553,9 @@ properties:
   ha_proxy.accept_proxy:
     description: "Turned off by default. Enforces the use of the PROXY protocol for all incoming connections to all frontends, with the exception of local requests to the health check endpoint, since Monit does not support PROXY protocol. When enabled, standard TCP connections to these ports no longer work."
     default: false
+  ha_proxy.expect_proxy:
+    description: "Turned off by default. Enables proxy protocol for the specified IP CIDR ranges. Only applies if `ha_proxy.accept_proxy` is disabled."
+    default: false
```

Comment:
this should be a list of the CIDRs that should use the `expect-proxy` directive in HAProxy, not a boolean.

Reply:
thanks